src/index.ts

```diff
@@ -21,35 +25,35 @@ export async function bundle(packageIdentifier: string, outputFilePath: string,
         let resultCode = undefined;
         await tempy.directory.task(async tempDirectory => {
             const saveFileFromPackage = createFetcher(cache.cacheFactory);
-            const pkgJsonData: any = await saveFileFromPackage(tempDirectory, packageName, packageVersion, 'package.json');
-            const pkgJson = JSON.parse(pkgJsonData);
+            const loadFileForPackage = (filePath: string, content?: string) => saveFileFromPackage(tempDirectory, packageName, packageVersion, filePath, content);
+            const typesEntryContent = await createPackageTypes(packageName, packageVersion, loadFileForPackage);
+            // load the generated file to the cache so it is
```

Comment:
so it is...? what? 🙃

Reply:
this is what you get when NOT using GPT...